src/Microsoft.AspNet.Razor.Runtime.Precompilation/Microsoft.AspNet.Razor.Runtime.Precompilation.xproj

```diff
@@ -0,0 +1,19 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Have to say, definitely prefer the precompilation in the Razor repo, mostly Razor specific bits. :+1: